src/main/java/frc/robot/constants/RobotMap.java

```diff
@@ -30,9 +30,13 @@
             public static final int REAR_RIGHT_ANGLE_ENCODER_ID = 10;
         }
 
-        public static class MOTOR_ID {
+        public static class IntakeOpener {
             public static final int INTAKE_OPENER_MOTOR_ID = 3;
         }
+
+        public static class Pitcher {
+            public static final int PITCHER_MOTOR_ID = 12;
```

Comment:
delete PITCHER_ in line 38

Reply:
but then we wont know were it comes from and the we would need to change the other ids

Reply:
Anything in pitcher belongs to the pitcher

Reply:
ok